packages/amplify-migration-e2e/src/gen1ResourceDetailsFetcher.ts

```diff
@@ -0,0 +1,43 @@
+import path from 'path';
+import { RefactorCategory } from './templategen';
+import { getProjectMeta } from '@aws-amplify/amplify-e2e-core';
+import { assertIdentityPool, assertStorage, assertUserPool, assertUserPoolClients } from './assertions';
+import { getResourceDetails } from './sdk_calls';
+
+async function getGen1AuthResourceDetails(projRoot: string) {
+  const gen1ProjRoot = path.join(projRoot, '.amplify', 'migration');
+  const gen1Meta = getProjectMeta(gen1ProjRoot);
+  const gen1Region = gen1Meta.providers.awscloudformation.Region;
+  const { gen1UserPoolId } = await assertUserPool(gen1Meta, gen1Region);
+  const { gen1IdentityPoolId } = await assertIdentityPool(gen1Meta, gen1Region);
+  const { gen1ClientIds } = await assertUserPoolClients(gen1Meta, gen1Region);
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [gen1ClientIdWeb, gen1ClientId] = gen1ClientIds;
```

Comment:
Please don't do this.

If this will be used earlier then this should be added later in follow up PR.
Otherwise if follow up never happens we're left with debt.

Reply:
Noted. I'll keep this in mind. [b28e101](https://github.com/aws-amplify/amplify-cli/pull/13993/commits/b28e1014fa504a64624b444c916fde099102071e)